webapp/src/blocks/block.ts

```diff
@@ -2,7 +2,10 @@
 // See LICENSE.txt for license information.
 import {Utils} from '../utils'
 
-type BlockTypes = 'board' | 'view' | 'card' | 'text' | 'image' | 'divider' | 'comment'
+const contentBlockTypes = ['text', 'image', 'divider'] as const
```

Comment:
I guess this is about type checking, but I think this is another place that you have to edit to make it work. Is there any way to not handle this cases and make it "dynamic".

Reply:
I can see dynamic plugins in the future, but for the short term, we expect extensions to happen in the code, and we should rely on static type checking as much as possible. Have to think about this more - maybe there will be a special "custom" type that has a dynamic "subtype" which is 3rd party extendable.

Reply:
Yes, as far as it gets caught by the type checker whenever y try to register the new component, works for me. That way, at least we have a explicit error that says "change this here".